consumers/src/main/java/uk/gov/moj/sdt/consumers/SubmitQueryConsumer.java

```diff
@@ -175,28 +171,14 @@ private SubmitQueryResponseType invokeTargetAppService(final SubmitQueryRequestT
         return null;
     }
 
-    /**
-     * @return the transformer for SubmitQueryConsumer
-     */
-    // CHECKSTYLE:OFF
-    public IConsumerTransformer<SubmitQueryResponseType, SubmitQueryRequestType, ISubmitQueryRequest, ISubmitQueryRequest>
-    getTransformer() {
-        return this.transformer;
-    }

```

Comment:
Again, wondering if it's worth keeping this in just in case.

Reply:
Same answer as above